### ISSUE TITLE
Support for Angular 5

### ISSUE THREAD

**Issue by philjones88** [MrWolfZ/ngrx-forms#13]
Get warnings when installing to an Angular 5 project

```
warning "ngrx-forms@1.1.1" has incorrect peer dependency "@angular/core@^4.0.0".
warning "ngrx-forms@1.1.1" has incorrect peer dependency "@angular/forms@^4.1.0".
```

**Comment by MrWolfZ** [comment]
@philjones88 I am currently working on version 2 of ngrx-forms and it's almost done. In that version I have fixed the peer dependency to require `>=4.00` of Angular. However, for now you can just ignore the error. I'll close this issue once version 2 is released.

**Comment by philjones88** [comment]
@MrWolfZ Thanks, current version seems to work fine with Angular 5 anyway.

**Comment by MrWolfZ** [comment]
I have just released version 2.0.0 which solves this issue.
